src/modules/modes/centurion/tapes/mix-tape.ts

```diff
@@ -57,6 +57,8 @@ export type FeedEvent = {
 } & (Horn | Song | Effect | Bpm | Other);
 
 export default interface MixTape {
+  /** Artist name, needs not to be unique */
+  artist: string;
   /** Unique name of the tape */
```

Comment:
This comment is no longer true, because the Gebroeders Scooter Centurion and the ATHENA Centurion both have the same title. This also breaks the initialization of Centurion Mode, as the title is used as the tape's name is used as primary key. Any reason why you want to separate these two fields?

Reply:
In the frontend tapes are now grouped by artist. To support this, I separated these fields in the backend as well (as this also made more sense to me than adding the artist name to the title).

When testing this in the frontend, this did not cause any issues, I even explicitly tested this. However, looking at the `enableCenturion` method I indeed see that this can give inconsistent behaviour.

My suggestion is to make the `artist` - `title` pair unique, and selecting the tape based on those parameters. If this is not expected to be unique together, we should maybe add an actual ID field.

Reply:
Whatever you think is nicest, I am fine with either implementation!

Reply:
I went for the easier option for now, and added artist as identifier for a tape.